.evergreen.yml

```diff
@@ -57,27 +82,67 @@ functions:
           curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh
   "package":
     - command: subprocess.exec
-      type: setup
+      type: test
       params:
         add_to_path:
-          - *go_bin
+          - ${go_bin}
           - ./bin
         working_dir: src/github.com/mongodb/mongocli
         env:
+          GOROOT: ${go_root}
           <<: *go_options
         command: make package
   "release":
     - command: subprocess.exec
-      type: setup
+      type: test
       params:
         add_to_path:
-          - *go_bin
+          - ${go_bin}
           - ./bin
         working_dir: src/github.com/mongodb/mongocli
         env:
           GITHUB_TOKEN: ${github_token}
+          GOROOT: ${go_root}
           <<: *go_options
         command: make release
+  "install go-msi":
+    - command: subprocess.exec
+      type: setup
+      params:
+        working_dir: src/github.com/mongodb/mongocli
+        command: choco install -y "go-msi" --force
+  "uninstall go-msi":
+    - command: subprocess.exec
+      type: setup
+      params:
+        working_dir: src/github.com/mongodb/mongocli
+        command: choco uninstall -y "go-msi"
+  "generate msi":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src/github.com/mongodb/mongocli
+        script: |
+          set -o xtrace
+          set -o errexit
+          set -o verbose
+
+          export GOROOT="c:\\golang\\go1.14"
+          export GOCACHE="$(cygpath --mixed "${workdir}\.gocache")"
+          export CGO_ENABLED=0
+          export GOMSI="/cygdrive/c/Program Files/go-msi"
+          export PATH="$GOMSI:/cygdrive/c/wixtools/bin:$GOROOT\bin:$PATH"
+
+          go-msi check-env
+
+          VERSION=$(git describe | cut -d "v" -f 2)
+
+          env GOOS=windows GOARCH=amd64 go build -ldflags "-s -w -X github.com/mongodb/mongocli/internal/version.Version=$VERSION" -o mongocli.exe
+
+          mkdir {build,dist}
+
+          go-msi make --msi "dist/mongocli_$VERSION.msi" --version $VERSION
+
```

Comment:
Wow! Having had to do some msi work in the past, that's a _lot_ easier in go than other configurations.

Reply:
well the hard part was making it run in evg